var/spack/repos/builtin/packages/serenity/package.py

```diff
@@ -0,0 +1,137 @@
+    depends_on("python@3.6:", when="+python", type=("build", "run"))
+    depends_on("py-pip", when="+python", type="build")
+    depends_on("py-pybind11", when="+python", type="build")
+    depends_on("serenity-libint")
+    depends_on("xcfun")
+
+    def patch(self):
+
+        filter_file(
+            "include(CMakeParseArguments)",
+            'message(FATAL_ERROR "Tried to download a dependency")',
+            "cmake/DownloadProject.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "find_package(GTest 1.8.1 QUIET)",
+            "find_package(GTest REQUIRED)",
+            "cmake/ImportGTest.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "find_package(GMock 1.8.1 QUIET)",
+            "return()",
+            "cmake/ImportGTest.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "function(import_libecpint)",
+            "function(import_libecpint)\n"
+            "find_package(ecpint CONFIG REQUIRED)\n"
+            "add_library(ecpint INTERFACE IMPORTED)\n"
+            "target_link_libraries(ecpint INTERFACE ECPINT::ecpint)\n",
+            "cmake/ImportLibecpint.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "function(import_libint)",
+            "function(import_libint)\n"
+            "find_package(Libint2 CONFIG REQUIRED)\n"
+            "add_library(libint2-static INTERFACE IMPORTED)\n"
+            "target_link_libraries(libint2-static INTERFACE Libint2::libint2)\n",
+            "cmake/ImportLibint.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "function(import_libxc)",
+            "function(import_libxc)\n"
+            "find_package(PkgConfig QUIET)\n"
+            "pkg_check_modules(pc_libxc libxc)\n"
+            "if(pc_libxc_FOUND)\n"
+            "add_library(xc INTERFACE IMPORTED)\n"
+            "target_link_libraries(xc INTERFACE ${pc_libxc_LINK_LIBRARIES})\n"
+            "target_include_directories(xc INTERFACE ${pc_libxc_INCLUDE_DIRS})\n"
+            "endif()",
+            "cmake/ImportLibxc.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "function(import_pybind11)",
+            "function(import_pybind11)\nfind_package(pybind11 REQUIRED)",
+            "cmake/ImportPybind11.cmake",
+            string=True,
+        )
+
+        filter_file(
+            "function(import_xcfun)",
+            "function(import_xcfun)\n"
+            "find_package(XCFun CONFIG REQUIRED)\n"
+            "add_library(xcfun INTERFACE IMPORTED)\n"
+            "target_link_libraries(xcfun INTERFACE XCFun::xcfun)\n",
+            "cmake/ImportXCFun.cmake",
+            string=True,
+        )
+
+    def cmake_args(self):
+        args = [
+            self.define("SERENITY_BUILD_TESTS", self.run_tests),
+            self.define("SERENITY_BUILD_PYTHON_BINDINGS", "+python" in self.spec),
+            self.define("SERENITY_MARCH", ""),
```

Comment:
What does this setting imply? Note that we'll inject `-march` or similar flags via the compiler wrapper.

Reply:
The SERENITY_MARCH / SCINE_MARCH argument will set some march option unless it is an empty string.

Reply:
Thanks for clarifying, all good then.